Gemfile

```diff
@@ -4,16 +4,16 @@ source 'http://rubygems.org'
 
 ruby File.read('.ruby-version').chomp
 
-gem 'css_parser'
-gem 'nokogiri'
-gem 'rails', '~> 6.1'
```

Comment:
I'm going to remove the requirement of Rails since not all systems use it. The Railtie is nice to have for systems with Rails, so I think it's still good to keep around.